### PR TITLE
Improve contribution instructions in README file

### PR DESCRIPTION
**Pull request by paul-tavares**
# Description

- Adds information on how to preview the documentation changes once the build is complete

image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,10 +9,20 @@ Please view this template for guidance on creating issues: https://github.com/el
...
 
 If you are an Elastic employee and would like to contribute to Elastic Security documentation: 
+
 1. Please clone and fork the `security-docs` repo. 
 2. Open an issue using the appropriate [template](https://github.com/elastic/security-docs/tree/master/.github/ISSUE_TEMPLATE).
 3. Check out the `main` branch and fetch the latest changes. 
 4. Check out a new branch and make your changes. 
 5. Save your changes and open a pull request. 
-6. Tag the the `@security-docs` team and any other appropriate reviewers. We'll take care of merging and backporting.  
+6. Tag the the `@security-docs` team and any other appropriate reviewers. We'll take care of merging and backporting.
```
Comment (paul-tavares) [inline README.md:18]: I think this item also needs updating. I'm not sure what `tag` means here. Is it a mention in the Description field? or a specific label (set of labels?)

File: README.md [hunk 2]
```diff
@@ -9,10 +9,20 @@ Please view this template for guidance on creating issues: https://github.com/el
...
 
 If you are an Elastic employee and would like to contribute to Elastic Security documentation: 
+
 1. Please clone and fork the `security-docs` repo. 
 2. Open an issue using the appropriate [template](https://github.com/elastic/security-docs/tree/master/.github/ISSUE_TEMPLATE).
 3. Check out the `main` branch and fetch the latest changes. 
 4. Check out a new branch and make your changes. 
 5. Save your changes and open a pull request. 
-6. Tag the the `@security-docs` team and any other appropriate reviewers. We'll take care of merging and backporting.  
+6. Tag the `@security-docs` team and any other appropriate reviewers. We'll take care of merging and backporting.
```
Comment (jmikell821) [inline README.md:18]: @paul-tavares just adding some more info for context. Nothing incorrect on your part! 

```suggestion
6. Add the `[@elastic/security-docs](https://github.com/orgs/elastic/teams/security-docs)` team and any other appropriate members as reviewers. 
7. Add the appropriate release version label, backport version label if appropriate, and team label to the PR. 
8. Once the docs team approves all changes, you can merge it. If a backport version label was added to a PR for stack versions 7.14.0 and newer, mergify will automatically open a backport PR. 
9. Merge the backport PR once it passes all CI checks.

**Review by jmikell821 — COMMENTED** [review]
@paul-tavares thanks for adding some extra info to this -- it's much appreciated! I just added a bit of more info since this PR is already open.

**Comment by paul-tavares** [comment]
Thank you @jmikell821 . Just applied your suggested changes :)

**Pull request merged**